src/AvaloniaEdit/CodeCompletion/CompletionList.xaml

```diff
@@ -1,13 +1,11 @@
 <Styles xmlns="https://github.com/avaloniaui"
-        
         xmlns:cc="clr-namespace:AvaloniaEdit.CodeCompletion;assembly=AvaloniaEdit"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        x:CompileBindings="False">
```

Comment:
Note that `x:CompileBindings="False"` is the reason why the completion list is empty. I am actually not sure why.

Reply:
Any ideas about this @Gillibald @maxkatz6 ?

Reply:
Not sure why either, but I assume some CLR members were trimmed as unused, since they were only referenced with non-compiled (reflection) bindings.